uuid.go

```diff
@@ -307,6 +307,11 @@ func (u *UUID) UnmarshalBinary(data []byte) (err error) {
 	return
 }
 
+// IsZero returns true if UUID is zero
```

Comment:
Nitpick. This comment's sentence doesn't end with a period. Other comments do, so this is slightly inconsistent.